### DISCUSSION TITLE
See which bank is used for each track

### DISCUSSION THREAD

**Discussion by industdev** [FluidSynth/fluidsynth#1179] [Q&A] [answered]
I have a sf2 file with more than 500 samples, I would like to know if there's a way for me to see which sample is used for each track from the midi file. 
(I want to play it in a DAW, loading a bank for each track of the midi file and I want to know which bank is used for which track)

There doesn't seem to be any command line argument for this.

**Comment by derselbst** [comment, marked as answer]
There is no way to see which sample(s) are in use from command line.

You can use the `channels` command to see which soundfont presets are active for each MIDI channel. Note that these are selected per MIDI _channel_ and not per MIDI track as your question suggests.
